Assets/Scripts/UI/BossTitle/TextWriter.cs

```diff
@@ -17,15 +17,20 @@ class TextWriter : MonoBehaviour
         private float secondsPerCharacter = .5f;
         [SerializeField]
         private string fullText; //The whole bazinga to type
+        public string FullText { set { fullText = value; } }
         [SerializeField]
-        private Util.SoundPlayer sound; //Sound that is played on each character
```

Comment:
Why did you remove this?  It is a wrapper around AudioSource, so using AudioSource directly isn't necessary and makes audio balancing harder.